### PR TITLE
Fixed an issue with id generation for the key manager

### PR DESCRIPTION
**Pull request by Mikkel-Boje**

### CODE REVIEW

File: client/src/key_management/key_manager.rs [hunk 1]
```diff
@@ -17,18 +16,19 @@ pub struct KeyManager {
...
+                (PreKey::Kyber, 0u32),
+                (PreKey::OneTime, 0u32),
+            ]),
         }
     }
 
     fn get_new_key_id(&mut self, key_type: &PreKey) -> u32 {
-        let id = self.key_incrementer_map.get_mut(key_type).unwrap().deref() + 1u32;
-        id - 1
+        let id = self.key_incrementer_map.get(key_type).unwrap().clone();
```
Comment (AOsnes) [inline client/src/key_management/key_manager.rs:29]: unwrap bad

Comment (Mikkel-Boje) [reply client/src/key_management/key_manager.rs:29]: Nope we know that this key exist in that it was created with the object

**Review by MJHC — DISMISSED** [review]
ok

**Review by MJHC — APPROVED** [review]
ok

**Pull request merged**